webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -275,6 +275,12 @@ object StringFormatter {
 
   def isKnoraOntologyIri(iri: SmartIri): Boolean =
     iri.isKnoraApiV2DefinitionIri && OntologyConstants.InternalOntologyLabels.contains(iri.getOntologyName)
+
+  def makeValueIri(resourceIri: IRI, uuid: String): IRI =
```

Comment:
*issue*: The uuid parameter is a String, in fact that String must be a base64 encoded

Do we really have to expose a plain `String` as a parameter? 
This would make it possible to create a new IRI for a value which is not following our conventions.
I think we should make this method here at least private if not remove it.

IMHO, later we should probably introduce a value class `ValueIri`.

Reply:
removed in https://github.com/dasch-swiss/dsp-api/pull/3271/commits/42b9784a2e40e51c84d810d02742663d53e298e0